cmd/oci-unpack/main.go

```diff
@@ -117,6 +118,9 @@ func (v *unpackCmd) Run(cmd *cobra.Command, args []string) {
 
 	case image.TypeImage:
 		err = image.Unpack(args[0], args[1], v.ref)
+
+	default:
+		err = errors.New("Error media type")
```

Comment:
I think it's good to say what it is that's not implemented (unpacking in this case) to distinguish among all the things that might not be implemented.